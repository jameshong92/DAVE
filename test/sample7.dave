// test if-elif-else
int main() {
	int a = 0;
    int b = 2;
    if (b == 0) {
        print("b is 0");
    } elif (a > 0) {
        print("a is larger than 0");
    } elif (b > 0) {
        print("b is larger than 0");
    } else {
        print("just a joke!");
    }
}