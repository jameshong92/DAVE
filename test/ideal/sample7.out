b is larger than 0